noria-server/mir/src/query.rs

```diff
@@ -71,9 +71,10 @@ impl MirQuery {
         mut self,
         table_mapping: Option<&HashMap<(String, Option<String>), String>>,
         sec: bool,
-    ) -> MirQuery {
+    ) -> (MirQuery, Vec<MirNodeRef>) {
```

Comment:
It'd be good to document the return type here as well.

Reply:
Done.